examples/graph/sdpa_bottom_right_causal_mask.cpp

```diff
@@ -0,0 +1,360 @@
+
+/*******************************************************************************
```

Comment:
Do you want to update the sdpa doc to include this example?

Reply:
Added.